packages/core/src/contrib/Minkowski.ts

```diff
@@ -192,3 +192,40 @@ export const rectangleSignedDistance = (
   // Return the signed distance
   return add(e1, ne2);
 };
+
+/**
+ * Constraint checking whether `point` is inside a convex polygon with vertices `polygonPoints`.
+ * @param polygonPoints Sequence of points defining a convex polygon.
+ * @param point Testing point.
+ * @param padding Padding applied to the polygon.
+ */
+const containsConvexPolygonPoints = (
+  p1: VarAD[][],
+  p2: VarAD[],
+  padding: VarAD
+): VarAD => {
+  const center = ops.vdiv(p1.reduce(ops.vadd), varOf(p1.length));
+  const sides = Array.from({ length: p1.length }, (_, i) => i).map((i) => [
```

Comment:
Had to look up how array-like objects work to understand this. For best readability (and potentially performance?), perhaps a simple loop would be better.

Reply:
Hopefully improved the readability a bit and added a clarifying comment. From some basic experiments, the performance should not be affected. It also does not matter here.